bitnami/orangehrm/requirements.yaml

```diff
@@ -1,5 +0,0 @@
-dependencies:
```

Comment:
It looks like we are removing this file, is it a mistake?

Reply:
It is desired, in Helm 3 the dependencies are set in the _Chart.yaml_, you can see mariadb as a dependency in that file right now

Reply:
Ok!